src/parameterizations/lateral/MOM_Zanna_Bolton.F90

```diff
@@ -141,9 +151,23 @@ subroutine ZB2020_init(Time, G, GV, US, param_file, diag, CS, use_ZB2020)
                  "subgrid momentum parameterization of mesoscale eddies.", default=.false.)
   if (.not. use_ZB2020) return
 
+  call get_param(param_file, mdl, "USE_ANN", CS%use_ann, &
```

Comment:
Could you make this parameter specific to ZB20, so that it does not conflict with ANNs in other modules? e.g. `ZB2020_USE_ANN`